packages/preview/octique/0.1.0/typst.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "octique"
+version = "0.1.0"
+entrypoint = "octique.typ"
+authors = ["0x6b"]
+license = "MIT"
+description = "GitHub [Octicons](https://primer.style/foundations/icons/) for Typst."
```

Comment:
I've missed this the first time around, but the description can only be plain-text, not Markdown.